README.md

```diff
@@ -18,7 +18,7 @@ Many of the guidelines make use of the header-only Guidelines Support Library. O
 
 ## Background and scope
 
-The aim of the guidelines is to help people to use modern C++ effectively. By "modern C++" we mean C++11, C++14, and C++17. In other
+The aim of the guidelines is to help people to use modern C++ effectively. By "modern C++" we mean C++11, C++14, C++17, and C++20. In other
```

Comment:
Perhaps `By "modern C++" we mean C++11 and newer` would be better?
That way this doesn't need to get updated every time a new version of the C++ standard is released, just when the base version is changed (e.g C++11 is no longer considered modern).

Reply:
Makes sense! 
```suggestion
The aim of the guidelines is to help people to use modern C++ effectively. By "modern C++" we mean C++11 and newer. In other
```